StratisCore.UI/src/app/app.module.ts

```diff
@@ -17,6 +17,7 @@ import { GenericModalComponent } from './shared/components/generic-modal/generic
 import { ApiService } from './shared/services/api.service';
 import { GlobalService } from './shared/services/global.service';
 import { ModalService } from './shared/services/modal.service';
+import { FeaturesService } from './shared/services/features.service';
```

Comment:
why do we need a features service if we already have the api service?

Reply:
Single responsibility.  We are in a position to add more feature-specific functionality if required.  The apiService is purely a service to use to make the raw API http calls.

Reply:
makes sense